pkg/api/component_report.go

```diff
@@ -431,7 +435,7 @@ func (c *componentReportGenerator) getTestDetailsQuery(allJobVariants apitype.Jo
 						ANY_VALUE(cm.jira_component_id) AS jira_component_id,
 						COUNT(*) AS total_count,
 						ANY_VALUE(cm.capabilities) as capabilities,
-						SUM(success_val) AS success_count,
+						SUM(adjusted_success_val) AS success_count,
 						SUM(flake_count) AS flake_count,
```

Comment:
The other problem we found is flake_count isn't a 0 or 1, it can be higher.

You could replace these with `SUM(CASE WHEN flake_count > 0 THEN 1 ELSE 0 END) AS flake_count` I think to get the right answer